go.mod

```diff
@@ -3,19 +3,19 @@ module github.com/gokrazy/tools
 go 1.18
 
 require (
-	github.com/breml/rootcerts v0.2.0
+	github.com/breml/rootcerts v0.2.6
```

Comment:
Are any changes to go.mod and go.sum actually necessary, or could you revert these?

Reply:
Some of them might be, unless you updated it to use the latest gokrazy/internal (for the hostname() change I PR'd).
Usually I never bother with manually fudging with `go.mod` and just run `go mod tidy`, which updates and fixes checksums all on its own whenever I have to touch dependencies.